machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -47,15 +47,39 @@ It is computed using the solutions from the L0 and L1 grids with the equation
 .. math::
     f_{h=0} = f_{L0} + \frac{f_{L0}-f_{L1}}{r^{\hat{p}} - 1}
 
-Grid refinement study with ADflow
----------------------------------
+Grid Refinement Study on Airfoils
+======================================
 
-1. Generate fine grid (L0) with :math:`N=2^n m + 1` nodes along each edge.
-2. Coarsen the L0 grid :math:`n-1` times using :code:`cgns_utils coarsen`.
-3. Use :code:`solveCL` or :code:`solveTrimCL` in ADflow to obtain :math:`C_D` for a given :math:`C_L`.
+There are two methods for performing grid refinement: 
+1) coarsening the volume mesh and 
+2) coarsening the surface mesh and extruding the family of surface meshes
+We discuss the pros and cons of each method and underlying theory;
+it is up to the user to choose the method.
+
+A few general notes first:
+   - With regards to aerodynamic shape optimization, being in the asymptotic regime is not always necessary since the objective function like drag is just offset by the truncation error.
+     This is not as simple with a coupled structural model.
+     As long as the CFD mesh is fine enough to capture the correct physical trends, then the design space will be accurate enough such that coarse mesh optimizations will get you close enough to the optimal solution;
+     subsequently, one can use finer meshes using the design variables from the coarser optimizations, thus decreasing overall computational cost.
+   - Redo your mesh convergence study on the optimized result to double check everything is behaving as expected
+   - Plotting contours of :math:`y^+`, shocks, and wake can help with debugging
+
+.. _option-1:
+
+Option 1: Coarsening volume meshes
+----------------------------------
+
+1. Generate fine grid (L0) with :math:`N=(2^n) (m) + 1` nodes along each edge.
+2. Coarsen the L0 grid :math:`n-1` times using ``cgns_utils coarsen``.
+3. Use ``solveCL`` or ``solveTrimCL`` in ADflow to obtain :math:`C_D` for a given :math:`C_L`.
```

Comment:
In the past I just kept the same fixed angle of attack and did a convergence study on both lift and drag, but this sounds a bit more rigorous for C_D so I am fine with having this as recommended approach if others are ok too

Reply:
Did we decide what approach to recommend? @sseraj your comment makes a lot of sense

Reply:
I think @gang525 and I agreed we should just remove this line

Reply:
Yeah I think mathematically you want to keep the same boundary conditions each time.